pkg/projects/Microsoft.NETCore.UniversalWindowsPlatform/project.json

```diff
@@ -0,0 +1,38 @@
+{
+  "dependencies": {
+    "Microsoft.NETCore": "5.0.1-rc3-24126-00",
+    "Microsoft.NETCore.Portable.Compatibility": "1.0.1-rc3-24126-00",
+    "Microsoft.NETCore.Runtime": "1.0.2-rc3-24126-00",
```

Comment:
NETCore.Runtime pulls in CoreCLR package? If so, will this package come from closed build for for UWP since that is what should happen?